pkg/agent/manager/manager.go

```diff
@@ -294,6 +297,13 @@ func (m *manager) runSynchronizer(ctx context.Context) error {
 			m.c.Log.WithError(err).Error("Synchronize failed")
 		default:
 			m.synchronizeBackoff.Reset()
+			syncInterval = m.synchronizeBackoff.NextBackOff()
```

Comment:
is not it going to result in never changing backoff?
syncInternval will no change when `err != nil` and  we are getting nextBackOff after reset, so syncInterval will never change

Reply:
Good catch @MarcosDY , I missed to add a call to NextBackOff() in the event of an error.